src/ray/stats/metric_defs.h

```diff
@@ -73,54 +60,41 @@ static Gauge ObjectStoreLocalObjects("object_store_num_local_objects",
-    "object_manager_unfulfilled_push_requests",
-    "Number of unfulfilled push requests for objects.", "requests");
-
-static Gauge ObjectManagerProfileEvents("object_manager_num_buffered_profile_events",
-                                        "Number of locally-buffered profile events.",
-                                        "events");
-
-static Gauge NumSubscribedTasks(
-    "num_subscribed_tasks",
-    "The number of tasks that are subscribed to object dependencies.", "tasks");
-
-static Gauge NumRequiredTasks("num_required_tasks",
-                              "The number of tasks whose output object(s) are "
-                              "required by another subscribed task.",
-                              "tasks");
-
-static Gauge NumRequiredObjects(
-    "num_required_objects",
-    "The number of objects that are required by a subscribed task.", "objects");
-
-static Gauge NumPendingTasks("num_pending_tasks",
-                             "The number of tasks that are pending execution.", "tasks");
-
-static Gauge NumPlaceableTasks(
-    "num_placeable_tasks",
-    "The number of tasks in the scheduler that are in the 'placeable' state.", "tasks");
-
-static Gauge NumWaitingTasks(
-    "num_waiting_tasks",
-    "The number of tasks in the scheduler that are in the 'waiting' state.", "tasks");
-
-static Gauge NumReadyTasks(
-    "num_ready_tasks",
-    "The number of tasks in the scheduler that are in the 'ready' state.", "tasks");
-
-static Gauge NumRunningTasks(
-    "num_running_tasks",
-    "The number of tasks in the scheduler that are in the 'running' state.", "tasks");
-
 static Gauge NumInfeasibleTasks(
     "num_infeasible_tasks",
     "The number of tasks in the scheduler that are in the 'infeasible' state.", "tasks");
+
+static Histogram HeartbeatReportMs(
+    "heartbeat_report_ms",
+    "Heartbeat report time in raylet. If this value is high, that means there's a high "
+    "system load. It is possible that this node will be killed because of missing "
+    "heartbeats.",
+    "ms", {100, 200, 400, 800, 1600, 3200, 6400, 15000, 30000});
+
+///
+/// GCS Server Metrics
+///
+static Count UnintentionalWorkerFailures(
+    "unintentional_worker_failures_total",
+    "Number of worker failures that are not intentional. For example, worker failures "
+    "due to system related errors.",
+    "worker_failures");
+
+static Count NodeFailureTotal("node_failure_total",
+                              "Number of node failures happened in the cluster.",
+                              "node_failures.");
+
+static Gauge PendingActors("pending_actors", "Number of pending actors in GCS server.",
+                           "actors");
+
+static Gauge PendingPlacementGroups("pending_placement_groups",
+                                    "Number of pending placement groups in GCS server.",
+                                    "placement_groups");
+
+static Histogram OutboundHeartbeatSizeKB("outboud_heartbeat_size_kb",
```

Comment:
```suggestion
static Histogram OutboundHeartbeatSizeKB("Outboud_heartbeat_size_kb",
```

Reply:
I think all the names are started from the lowercase? Btw, I noticed it was `outbound` not `outboud`, so I will fix that instead